src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -156,6 +245,113 @@ public abstract partial class JsonNamingPolicy
         public static System.Text.Json.JsonNamingPolicy CamelCase { get { throw null; } }
         public abstract string ConvertName(string name);
     }
+    public partial class JsonNode
+    {
+        public JsonNode() { }
+    }
+    public partial class JsonNull : System.Text.Json.JsonNode
+    {
+        public JsonNull() { }
+    }
+    public partial class JsonNumber : System.Text.Json.JsonNode
+    {
+        public JsonNumber() { }
+        public JsonNumber(bool value) { }
```

Comment:
I don't think this is right, how would you construct a number based on a bool?

Reply:
I just thought to convert it to 0/1, but probably if we have `JsonBool` already, it doesn't make sense.

Reply:
yeah I don't think that this will be very useful, I would just remove it.